tests/opwen_email_server/utils/test_email_parser.py

```diff
@@ -13,6 +14,13 @@
     'files', 'opwen_email_server', 'utils', 'test_email_parser'))
 
 
+def _given_test_image_base64(size):
```

Comment:
Here you're relying on the caller to always pass either `small` or `large` as the string argument. You can enforce this using the enum feature: https://docs.python.org/3/library/enum.html

This is how you'd declare all the possible image sizes:

```py
from enum import Enum
from enum import auto
from enum import unique


@unique
class ImageSize(Enum)
    large = auto()
    small = auto()
```

Then you can type annotate the function to ensure that we're always passing in a correct enum value:

```py
def _given_test_image_base64(size: ImageSize):
```

Finally, to get the value of the enum for looking up the key in the json, you can use the `.name` property:

```py
return json_images[size.name]
```

Other languages like Typescript have a very nice feature where they can say that "this variable is either string a or string b" like `size: "large"|"small"` but unfortunately Python doesn't support this so we have to use the slightly more verbose/explicit enum feature.